packages/desktop-client/src/components/SidebarWithData.js

```diff
@@ -79,8 +79,8 @@ function EditableBudgetName({ prefs, savePrefs }) {
     return (
       <Button
         type="bare"
-        color={colors.n9}
         style={{
+          color: theme.sidebarItemText,
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏the color is wrong on hover

<img width="280" alt="Screenshot 2023-08-12 at 09 19 46" src="https://github.com/actualbudget/actual/assets/886567/f36436f7-3510-4daf-88d8-648212e02c57">

Reply:
Fixed